highfidelity/hifi
# fix issue with not seeing your own attachments

This fixes the bug where you can't see your own attachments.


This reverts some code that was part of this commit: https://github.com/highfidelity/hifi/pull/10255/commits/4511156d7bbba1cd558223eb14a10b6a2e21470f

@AndrewMeadows - was there a reason you made this change?

**Test Plan**
1. login
2. Attach something to yourself
3. Verify that you can see the attachment.
4. Verify that others can see the attachment.
